views/library/unified_search_list.html

```
@@ -1,154 +1,116 @@
-                hx-swap="innerHTML">
-          Next
-        </button>
-     </div>
-      <div class="text-sm text-gray-500 dark:text-gray-400">
-        <span class="hidden md:inline">Showing </span>{{.Pagination.Page}}/{{.Pagination.TotalPages}} <span class="hidden md:inline">pages </span>({{.Pagination.TotalCount}} total results)
-      </div>
-   </div>
+{{define "pagination_bar"}}
+{{if .Pagination}}{{if gt .Pagination.TotalPages 1}}
+<div class="flex items-center justify-between px-1 py-1">
+  <button class="px-3 py-1.5 rounded-md text-sm transition-colors {{if not .Pagination.HasPrev}}text-gray-400 dark:text-gray-600 cursor-not-allowed{{else}}text-gray-700 dark:text-gray-300 bg-gray-100 dark:bg-neutral-800 hover:bg-gray-200 dark:hover:bg-neutral-700{{end}}"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)